en/drafts/originals/understanding-creating-word-embeddings.md

```diff
@@ -165,19 +157,17 @@ from gensim.models import Word2Vec          # to access Gensim's flavor of Word2
-
-The code block below reads in the path to your corpus, iterates through that folder, pulls the text from each file, and stores it in a dictionary. This code is written to process a folder with plain text files (.txt). These files can be anywhere within this folder, including in sub-folders. This lesson assumes that you are working with the sample corpus, but the code can also be adjusted to reflect a different corpus.
-
-#### Coding tips for processing text files
+You might need to install some of the libraries used in this tutorial. See [this lesson](https://programminghistorian.org/en/lessons/installing-python-modules-pip) for more information on installing external Python libraries.
 
-A few important things to note:
+Now that we have our libraries, we need to load our data into Python. You will need to store your dataset locally on your computer, ideally somewhere easy to navigate to. For instance, you could place your data in a folder inside your `Documents` folder, or in the same repository as your code file. In either case, you will need to know the file path for that folder.
 
-1. When you are inputting your file path, you should use the _entire_ file path. For example, on a Windows computer, that file path might look something like: "C:/users/admin/Documents/MY_FOLDER". On a Mac, the file path might be: "/Users/admin/Documents/MY_FOLDER".
+The code block below reads the path to your corpus, iterates through that folder, pulls the text from each file, and stores it in a dictionary. You will provide the file path for the sample [corpus](https://github.com/ViralTexts/nineteenth-century-recipes) which you should have downloaded from Github (the code can also be adjusted to reflect a different corpus). When you are inputting your file path, you should use the entire file path. For example, on a Windows computer, that file path might look something like: `C:/users/admin/Documents/MY_FOLDER`. On a Mac, the file path might be: `/Users/admin/Documents/MY_FOLDER`. Your files can be anywhere within this folder, including sub-folders: the code will loop through the folders and sub-folders in the filepath provided and look for all `.txt` files. (You can use a file path to a folder full of different types of files, but this code is only going to look for `.txt` files.)
 
-2. You can use a file path to a folder full of different types of files, but this code is only going to look for _.txt files_. If you want to work with different file types, you'll have to change the "endswith(.txt)" call. However, keep in mind that these files should always contain some form of plain text. For example, a Word document (.doc) or a PDF won't work with this code.
+<div class="alert alert-info">
+If you want to work with different file types, you'll have to change the `endswith(.txt)` call. However, keep in mind that these files should always contain some form of plain text. For example, `.doc` or `pdf` won't work with this code. 
```

Comment:
I added this Information alert box to notify readers about the `endswith(.txt)` code they should change if they want to work with a different file format. However, I found the following part a little unclear: "keep in mind that these files should always contain some form of plain text. For example, `.doc` or `pdf` won't work with this code."
What do "these files" and "this code" refer to?